main.tf

```diff
@@ -0,0 +1,127 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = ">= 3.8"
+    }
+  }
+  backend "azurerm" {
+        resource_group_name  = "Cohort28_AleTan_ProjectExercise"
+        storage_account_name = "alexstorageex13"
+        container_name       = "bacpac1"
```

Comment:
Probably not the best name for this container, but reasonable to re-use the one from this workshop I suppose!

Reply:
Done :)